src/refactoring_guru/strategy/example/strategies/PayByPayPal.java

```diff
@@ -35,7 +35,7 @@ public void collectPaymentDetails() {
                 System.out.print("Enter password: ");
                 password = READER.readLine();
                 if (verify()) {
-                    System.out.println("Data verification was successful");
+                    System.out.println("Data verification has succeeded");
```

Comment:
to match success message in Demo class

Reply:
Then it should be "has been". Can you please fix it there too?